test/system/090-events.bats

```diff
@@ -25,3 +25,35 @@ load helpers
     run_podman events --filter type=container --filter container=$cname --filter event=start --stream=false
     is "$output" "$expect" "filtering just by label"
 }
+
+@test "image events" {
+    # `remove events` seem to be broken...
```

Comment:
Suggestion: move the quote, to make this "# 'remove' events seem". That makes it a little less ambiguous: with the quotes as they are, and the `skip` immediately below, my first thought was "is that a typo for 'remote'?"

Even better: "'remove' events under podman-remote seem ..."